tests/unit/capi/GEOSDensifyTest.cpp

```diff
@@ -0,0 +1,234 @@
+//
+// Test Suite for C-API GEOSBuffer and GEOSBufferWithStyle
+
+#include <tut/tut.hpp>
+// geos
+#include <geos_c.h>
+// std
+#include <cstdarg>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+
+namespace tut {
+//
+// Test Group
+//
+
+// Common data used in test cases.
+struct test_capigeosdensify_data {
```

Comment:
Can reduce boilerplate by including "capi_test_utils.h" and inheriting from `capitest::utility`. See `GEOSBoundaryTest` for an example.